src/main/java/de/qabel/desktop/QblMain.java

```diff
@@ -139,6 +139,9 @@ private QblMain() {
 		dropActorThread = new Thread(dropActor, "DropActor");
 		dropActorThread.start();
 		moduleManager = new ModuleManager(emitter, resourceActor);
+		Thread restServer = new Thread(new QblRESTServer(9696, resourceActor, dropActor, moduleManager));
```

Comment:
This is just a bit of nitpicking here, but would you mind to put the ```Thread restServer``` variable on top to the rest of the threads? Just to keep it consistent.